service/src/test/java/eu/europa/ec/fisheries/uvms/activity/fa/dao/FishingActivityDaoTest.java

```diff
@@ -50,38 +48,13 @@ public void prepare() {
     @Test
     @SneakyThrows
     public void findEntityById() {
-        // Given
```

Comment:
I don't know, I kind of like having all three comments present, even when they don't "contain" anything...